test/tests/multiapps/loose_couple_time_adapt/gold/begin.e

```diff
@@ -0,0 +1 @@
+end.e
```

Comment:
This is interesting. Are Exodus files not binary?

Reply:
This is actually a symlink, since both tests should yield the exact same exodus results. This works fine, but if we want real exodus files for both tests I can do that.

Reply:
Symlinks are fine. I just was not able to tell it was a symlink from the diff 👍